src/app/personside/infotabs/meldinger/traadliste/TraadListe.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { Traad } from '../../../../../models/meldinger/meldinger';
+import { AlertStripeInfo } from 'nav-frontend-alertstriper';
+import { genericDescendingDateComparator } from '../../../../../utils/dateUtils';
+import TraadListeElement from './TraadListeElement';
+import styled from 'styled-components';
+import theme from '../../../../../styles/personOversiktTheme';
+
+interface Props {
+    traader: Traad[];
+    oppdaterValgtTraad: (traad: Traad) => void;
+    valgtTraad?: Traad;
+}
+
+const PanelStyle = styled.div`
+    ${theme.hvittPanel};
+    min-width: 24rem;
+    flex-basis: 24rem;
+    ol {
+        list-style: none;
+    }
+`;
+
+const TraadListeStyle = styled.ol`
+    > * {
+        border-top: ${theme.border.skille};
+    }
+`;
+
+function SortertListe(props: Props) {
+    const sortertPåDato = props.traader.sort(genericDescendingDateComparator(traad => traad.dato));
```

Comment:
Kunne godt one-linet denne tenker jeg?

Reply:
Med linja under mener jeg

Reply:
Føler ikke `sortertPåDato` som et mellomsteg gir nødvendig verdi

Reply:
That said kan du godt rename den unødvendige lange engelske comparatoren

Reply:
Jeg synes det gir verdi 💰 ❤️